src/server.rs

```diff
@@ -280,22 +280,40 @@ impl Server {
         function: FunctionInfo,
         is_const: bool,
     ) {
+
+        self.save_route(py, route_type, route, &function, is_const);
+
+        if &route[route.len() - 1..] == "/" {
+            self.save_route(py, route_type, &route[0..route.len() - 1], &function, is_const);
+        } else {
+            self.save_route(py, route_type, &format!("{}/", route), &function, is_const);
+        }
+    }
+
+    pub fn save_route(
```

Comment:
also, why rename this function?

Reply:
> um... that's a new function. we're calling `#save_route` (the renamed function with the old code) twice from the new function with the old name (`#add_route`). I did it this way by renaming to avoid changing it's implementation from everywhere else.